### DISCUSSION TITLE
How to change the compiled app icon?

### DISCUSSION THREAD

**Discussion by toggery** [cawa-93/vite-electron-builder#247] [Q&A] [answered]
I'm working on windows, after run the script `npm run compile`, the app(.exe) shows the default icon in the windows explorer, how to change it?

**Comment by cawa-93** [comment, marked as answer]
https://www.electron.build/icons.html#windows-nsis
https://github.com/cawa-93/vite-electron-builder/tree/main/buildResources

**Reply by toggery** [reply]
I did it! Thanks for your patient guidance.
